erpnext/accounts/report/cash_flow/cash_flow.py

```
@@ -11,8 +11,7 @@
 from pypika import Order
 
 from erpnext.accounts.doctype.financial_report_template.financial_report_engine import (
-	FinancialReportEngine,
-	get_xlsx_styles,  #! DO NOT REMOVE - hook for styling
+	FinancialReportEngine,  #! DO NOT REMOVE - hook for styling
```

Ruff (0.15.15)

[warning] Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)